ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -308,6 +308,21 @@ aws|aws-arm64|aws-usgov)
       inject_spot_instance_config ${dir}
     fi
     ;;
+  gcp)
+    if [ X"${WORKER_RHCOS_IMAGE_FROM_GCP_MARKETPLACE}" != X"" ]; then
```

Comment:
Suggest to introduce a separate step like `/ipi/conf/gcp/marketplace` for it.

Reply:
It has to be after `create manifests` and before `create cluster`, so it looks like no way to insert an individual step with the current `ipi-install-install` implementation. Right?

Reply:
You are right.

As our discussed on slack, now we need to re-estimate if it is good time to integrate this testing in prow ci, let us wait for double confirm with devs and PMs for the user story.